packages/client/src/store/index.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	createReduxStore,
+	dispatch,
+	register,
+	resolveSelect,
+} from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import reducer from './reducer';
+import * as actions from './actions';
+import * as selectors from './selectors';
+import * as resolvers from './resolvers';
+import { ENTITY_KIND, ENTITY_NAME, STORE_NAME } from './constants';
+import { __ } from '@wordpress/i18n';
+import { store as coreStore } from '@wordpress/core-data';
+// Create a global variable to track if the store has been registered, this ensures we only register the store once across all imports
+// if multiple plugins are using the Feature API.
+// TODO: We may want to expose the api over wp.featureApi.* in the future like WordPress does.
+declare global {
+	interface Window {
+		__WP_FEATURE_API_STORE_REGISTERED?: boolean;
+	}
+}
+
+const isStoreRegistered = () => {
+	return window.__WP_FEATURE_API_STORE_REGISTERED === true;
+};
+
+export const store = createReduxStore( STORE_NAME, {
+	reducer,
+	actions,
+	selectors,
+	resolvers,
+} );
+
+if ( ! isStoreRegistered() ) {
+	try {
+		register( store );
+		window.__WP_FEATURE_API_STORE_REGISTERED = true;
+
+		// get registered features
+		resolveSelect( STORE_NAME ).getRegisteredFeatures();
+
+		// get registered server features
```

Comment:
This is not getting registered server features it is additing an enptity so we can use wp/date to get the server features.